### PR TITLE
[9.0][ADD] Backport of module pos_timeout

### PR DESCRIPTION
**Pull request by apineux**

### CODE REVIEW

File: pos_timeout/README.rst [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+To configure this module, you need to:
+
+* Go to 'Point Of Sale' / 'Configuration' / 'Point of Sale' and edit your
+  PoS Config, setting a timeout
+
+.. figure:: /pos_timeout/static/description/pos_config.png
+   :alt: PoS Configuration
+   :width: 800 px
+
+If not set, the default Odoo timeout will be used. (7.5 seconds in V10.0)
```
Comment (legalsylvain) [inline pos_timeout/README.rst]: how many in V9 ? the same ?

File: pos_timeout/README.rst [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+
+Usage
+=====
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/121/10.0
```
Comment (legalsylvain) [inline pos_timeout/README.rst]: :target: https://runbot.odoo-community.org/runbot/121/10.0

> 

    :target: https://runbot.odoo-community.org/runbot/121/9.0

**Review by legalsylvain — CHANGES_REQUESTED** [review]
Hi @apineux. thanks for your contribution, and welcome in OCA/pos.
I see that you did'nt cherry picked the commits, that is a convention in the OCA. could you do that ? 
Ref : https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-12.0#technical-method-to-migrate-a-module-from-110-to-120-branch
It makes the review easier to do.

kind regards.

File: pos_timeout/README.rst (hunk 1 above)
Comment (apineux) [reply pos_timeout/README.rst]: yes it is also 7.5 sec

**Comment by apineux** [comment]
Hello @legalsylvain 
Thank you for your review. I updated the PR to follow the OCA convention.

**Comment by nilshamerlinck** [comment]
Hi @legalsylvain @apineux 

Great module!

However, it replaces a dynamic value by a static value because of [this line on the native side](https://github.com/odoo/odoo/blob/9.0/addons/point_of_sale/static/src/js/models.js#L787):

```javascript
var timeout = typeof options.timeout === 'number' ? options.timeout : 7500 * orders.length;
```

Would you be open to introducing a `* orders.length` to keep it consistent with the native behavior (and avoid surprises after long disconnections)?

**Comment by nilshamerlinck** [comment]
Hi @legalsylvain here's the PR for that change: https://github.com/OCA/pos/pull/439

**Pull request closed without merging**